batchflow/research/named_expr.py

```diff
@@ -3,15 +3,15 @@
 import os
 
 from .results import Results
-from ..named_expr import NamedExpression
+from ..named_expr import NamedExpression, eval_expr
 
-class ResearchNamedExpression(NamedExpression):
+class ResearchNamedExpression(NamedExpression): #ResearchNamedExpression
```

Comment:
`#ResearchNamedExpression` Why?

Reply:
For consistency :) OK, I will remove it

Reply:
There is also an inconsistent number of spaces after `#` :-)